assets/src/edit-story/components/library/panes/media/karma/mediaFetching.karma.js

```diff
@@ -26,36 +26,54 @@ import { waitFor } from '@testing-library/react';
 import { Fixture, MEDIA_PER_PAGE } from '../../../../../karma/fixture';
 import { ROOT_MARGIN } from '../mediaPane';
 
-describe('MediaPane fetching', () => {
-  let fixture;
+const flagValues = [false, true];
+flagValues.forEach((value) => {
+  describe('MediaPane fetching', () => {
```

Comment:
Add 'rowBasedGallery' somewhere in the description, so that each of the 2 tests have a unique description.

Reply:
eg. something like

```
`MediaPane fetching for ${value ? 'row' : 'column'} gallery`
```

That saves you having to add this to each 'it'.